third_party/terraform/tests/resource_container_cluster_test.go.erb

```diff
@@ -2642,6 +2644,12 @@ resource "google_container_cluster" "with_autoprovisioning" {
       resource_type = "memory"
       maximum       = 2048
     }
+    auto_provisioning_defaults {
```

Comment:
What happens if I include `cluster_autoscaling` without `auto_provisioning_defaults` or `auto_provisioning_defaults` without `oauth_scopes`? Will I get a diff?

Reply:
Also, can you include an `_updated` config including `monitoring.write` and check that there are no changes in plan? (I think that's one of the options we have for a test step)

Reply:
I am adding O+C to `auto_provisioning_defaults` which should get rid of the diff if `auto_provisioning_defaults` is not included in a `cluster_autoscaling` block.